website/docs/partials/_jwt-generation-partial.md

```diff
@@ -18,7 +18,7 @@ The HTTP connection between your beacon node and execution node needs to be auth
 
 ```
 ## Optional. This command is necessary only if you've previously configured USE_PRYSM_VERSION
-SET USE_PRYSM_VERSION=v3.1.2
+SET USE_PRYSM_VERSION=<current prysm version>
```

Comment:
I think we should leave this with some kind of version string. Maybe use some templating to load the latest version string value? it might not be clear to the user what is supposed to go here imo